_ext/panels_fairplus.py

```diff
@@ -140,75 +144,92 @@ def _clean_options(self):
 
         self.options["intended_audience"] = list_of_intended_audiences
 
-        
+
         # has_executable_code
         assert self.options["has_executable_code"] in CONTROLLED_VOCABULARY_EXECUTABLE_CODE, \
             sphinx.errors.ExtensionError(_make_string_red(
                 f"The value of has_executable_code has to be out of the following controlled vocabulary: {', '.join(list(CONTROLLED_VOCABULARY_EXECUTABLE_CODE))} ."
                 ))
-            ## no joke, because "True" and "yes" are evaluated by YAML to be "", we need "yeah" and "nope"... 
+            ## no joke, because "True" and "yes" are evaluated by YAML to be "", we need "yeah" and "nope"...
 
+        assert ':' not in self.options["recipe_name"], \
+            sphinx.errors.ExtensionError(
+                _make_string_red("Do not use semicolon in recipe names %s" % (self.options["recipe_name"])))
```

Comment:
I propose to have the error message as : "The colon (i.e. the character ":") is not allowed in recipe_name: %s"

Reply:
@rgiessmann agreed. thx!